src/Language/TOML/Processing.idr

```diff
@@ -0,0 +1,92 @@
+-- SPDX-FileCopyrightText: 2022 The toml-idr developers
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module Language.TOML.Processing
+
+import Data.List.Quantifiers
+import Language.TOML.Value
+
+-- On Idris 0.5.2 release, remove this in favour of using Data.Singleton.
```

Comment:
I don't think this is needed anymore, since you removed the `Singleton` code.
```suggestion

```

Reply:
Oh good catch, yeah that comment is no longer needed.